vita3k/gui/src/settings_dialog.cpp

```diff
@@ -764,6 +769,10 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Uncheck the box to disable the display of the compile shaders dialog.");
         ImGui::Spacing();
+        ImGui::Checkbox("Show Touchpad Cursor", &config.show_touchpad_cursor);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Uncheck the box to disable show touchpad cursor on screen.");
```

Comment:
To disable something, don't put a verb after disable (except using the gerund form)

Reply:
"Uncheck to disable showing the touchpad cursor on-screen." (just putting this here since this is probably what it should say instead)

Reply:
> "Uncheck to disable showing the touchpad cursor on-screen." (just putting this here since this is probably what it should say instead)

thx, i just keep the box, for get same all other.